### PR TITLE
fix: bug in TeamSearch in Admin panel

### LINKED ISSUE #12982: Admin dashboard - team list cannot be searched by team slug
I was unable to search for "gitpod-ubp-test-team", but I _was_ able to search for "Gitpod UBP" or "UBP test team".

### PR DESCRIPTION
**Pull request by Siddhant-K-code**
> Signed-off-by: Siddhant Khare <siddhant@gitpod.io>

## Description
After Proposed changes, working with `-` also:

image

image

## Related Issue(s)

Fixes #12982 

## Release Notes

```release-note
NONE
```

## Documentation

No

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`

### CODE REVIEW

File: components/dashboard/src/admin/TeamsSearch.tsx [hunk 1]
```diff
@@ -98,7 +98,7 @@ export function TeamsSearch() {
                             placeholder="Search Teams"
                             onKeyDown={(k) => k.key === "Enter" && search()}
                             onChange={(v) => {
-                                setSearchTerm(v.target.value.trim());
+                                setSearchTerm(v.target.value.trim().replace(/-/g, " "));
```
Comment (Siddhant-K-code) [inline components/dashboard/src/admin/TeamsSearch.tsx:101]: But, Now this case will start failing 😞 

image

**Comment by svenefftinge** [comment]
[The issue ](https://github.com/gitpod-io/gitpod/issues/12982) is about also searching the team slug not replacing dashes in the search.

**Comment by Siddhant-K-code** [comment]
> [The issue ](https://github.com/gitpod-io/gitpod/issues/12982) is about also searching the team slug not replacing dashes in the search.

I think, along with `team.name`, we can also pass `team.slug`

**Comment by Siddhant-K-code** [comment]
if we add `team.slug` as OR clause to search by team by `slug` also

https://github.com/gitpod-io/gitpod/blob/7c0ecdbf86a3554a59770fc61d75f0a48f0336b0/components/gitpod-db/src/typeorm/team-db-impl.ts#L53

**Pull request closed without merging**